docs/docs/get_started/installation.mdx

```diff
@@ -22,7 +22,7 @@ A lot of the value of LangChain comes when integrating it with various model pro
 By default, the dependencies needed to do that are NOT installed.
 However, there are two other ways to install LangChain that do bring in those dependencies.
 
```

Comment:
We're actually planning on deprecating this extras group, i think we probably just want to delete this part of the docs. cc @eyurtsev

Reply:
@baskaryan Thanks for the review.
Which part of the docs do I need to remove?

Reply:
updated!